modules/gapi/src/backends/cpu/gcpubackend.cpp

```diff
@@ -140,6 +154,22 @@ cv::GArg cv::gimpl::GCPUExecutable::packArg(const GArg &arg)
     }
 }
 
+void cv::gimpl::GCPUExecutable::setupKernelState(const ade::NodeHandle &kernelNode,
+                                                 GArg &state)
+{
+    GConstGCPUModel gcm(m_g);
+    GCPUKernel k = gcm.metadata(kernelNode).get<Unit>().k;
+
+    k.m_setupF(GModel::collectInputMeta(m_gm, kernelNode),
+               m_gm.metadata(kernelNode).get<Op>().args,
+               state);
+}
```

Comment:
I think it doesn't make much sense here in a function that handles only one kernel, and rewrite it to call this callback for _all_ affected nodes. Then the caller code becomes more clear.

Reply:
This function is used twice in this code

Reply:
Done.